src/main/java/com/epam/esm/service/exception/ServiceException.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.esm.service.exception;
+
+import com.epam.esm.dao.exception.ErrorCodeEnum;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(value = {"stackTrace", "cause", "suppressed", "localizedMessage"}, allowSetters = true)
+public class ServiceException extends Exception {
```

Comment:
Please, keep only one custom exception

Reply:
Deletes Service exception, now Persistence exceptions serves it role